services/app-web/src/pages/Settings/SettingsTables/DivisionAssignmentTable.tsx

```diff
@@ -152,17 +150,23 @@ function CMSUserTableWithData({
             dateRangeFilter: () => true,
         },
         columns,
+        defaultColumn: {
+            size: 250,
+        },
         getCoreRowModel: getCoreRowModel(),
     })
 
     return (
-        <Table>
-            <caption className="srOnly">CMS Users</caption>
-            <thead className={styles.header}>
+        <Table bordered fullWidth className={styles.table}>
```

Comment:
Styles nit:
- This table has a `<p>` description like the others looking at the figma. 
- Also this table and the other tables have a heading that is a large font size but is not bold. Let's make sure we clean that up it effects the polish quite a bit I'm noticing.

Current:
<img width="957" alt="Screenshot 2024-09-05 at 10 11 11 AM" src="https://github.com/user-attachments/assets/6371a434-bf91-4546-900b-c90fd98a62d7">

Expected:
<img width="650" alt="Screenshot 2024-09-05 at 10 08 07 AM" src="https://github.com/user-attachments/assets/d4a96bcd-057e-4ee0-a72e-9e7f5ec6a8bd">